author: Brian Nelson <pyro@debian.org>

From Roman Zippel <zippel@linux-m68k.org>: These defines are taken from the
m68k tree, they should appear like this soon also in the upstream source.

--- a/src/corelib/io/qfilesystemwatcher_inotify.cpp
+++ b/src/corelib/io/qfilesystemwatcher_inotify.cpp
@@ -112,6 +112,10 @@
 # define __NR_inotify_init	240
 # define __NR_inotify_add_watch	241
 # define __NR_inotify_rm_watch	242
+#elif defined (__mc68000__)
+# define __NR_inotify_init	284
+# define __NR_inotify_add_watch	285
+# define __NR_inotify_rm_watch	286
 #else
 # error "This architecture is not supported. Please talk to qt-bugs@trolltech.com"
 #endif
